discovery-provider/src/solana/anchor_program_indexer.py

```diff
@@ -170,8 +308,12 @@ async def fetch_cid_metadata(
                             ]
 
                         # TODO update once user_id changes are merged in
+                        # NOTE: for update user, the user_id is not passes as an arg, so
```

Comment:
we're going to make this consistent with CreateUser and UpdateReplicaSet

Reply:
https://linear.app/audius/issue/ASI-1049/add-id-and-bump-to-updateuser-in-program